File: 0001 Code Chef/0021 Biryani classes/implementation/0005 kotlin/BiryaniClasses.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput != null){
                    var stringArray:Array<String> = stringInput.split(" ").toTypedArray();
                    if(stringArray.size == 2){
                        var x:Int? = stringArray[0].toIntOrNull();
                        var y:Int? = stringArray[1].toIntOrNull();
                        if(x != null && y != null){
                            var total:Int = x* y;
                            println(total);
                        }
                    }
                }
                count = count - 1;
            }
        }
    }
}

File: 0001 Code Chef/0021 Biryani classes/implementation/0003 c#/BiryaniClasses.cs
using System;

class BiryaniClasses{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            string[] stringArray = stringInput.Split(" ");
            int x = Convert.ToInt32(stringArray[0]);
            int y = Convert.ToInt32(stringArray[1]);
            int total = x * y;
            Console.WriteLine(total);
            test = test -1;
        }
    }
}